openmetadata-ui/src/main/resources/ui/src/components/common/ProfilePicture/ProfilePicture.tsx

```diff
@@ -61,12 +64,17 @@ const ProfilePicture = ({
   const getAvatarByName = () => {
     return (
       <Avatar
-        className={className}
-        height={height}
-        name={getEntityName({ name, displayName } as EntityReference)}
-        textClass={textClass}
-        type={type}
-        width={width}
+        className={classNames('flex-center', className)}
+        data-testid="profile-avatar"
+        icon={character}
+        shape={type}
+        size={size || parseInt(width)}
+        style={{
```

Comment:
not a fan of using style, If possible use class.

Reply:
since there are font and background colors that are generated via userName, so the inline style is more efficient to use here rather than classes!